docs/operation/ros_interface/ros2.rst

```diff
@@ -62,6 +62,13 @@ Parameters
 
 The **slate_base_node** has the following parameters:
 
+* ``update_frequency``:
+
+  * **Type**: int
+  * **Description**: The frequency in Hz of the SLATE update loop.
+  This is the rate at which the SLATE updates based on the incoming data.
+  * **Default**: ``10``
```

Comment:
Default is 20: https://github.com/Interbotix/interbotix_ros_core/blob/af18d4fe24ba08e09a0f1e92afaca1863e3205de/interbotix_ros_slate/interbotix_slate_driver/src/slate_base.cpp#L45
```suggestion
  * **Default**: ``20``
```